addons/sourcemod/scripting/shop.sp

```diff
@@ -1124,10 +1132,6 @@ public int ItemPanel_Handler(Menu menu, MenuAction action, int param1, int param
 							CPrintToChat(param1, "%t", "EmptyInventory");
 						}
 					}
-					else
-					{
-						ShowItemInfo(param1, iClItemId[param1]);
-					}
```

Comment:
Uhm... strange block of code, you confirming sell or showing item info, and this menu can be overlapped by **Main menu** by lines [1127-1134](https://github.com/FD-Forks/Shop-Core/pull/118/files#diff-e445d29076c21bd33405c776495e44e9766a041a3f2b32c9c49c8e2fa8b177eeR1127-R1134)